### ISSUE TITLE
Merge master and remote-json branches

### ISSUE THREAD

**Issue by Akahadaka** [goodwilltyping/OpenFight#2]
I suggest attempting to duplicate the **look and feel** of the _remote-json_ branch into master without merging, but rather an exercise in HTML and CSS. **Ignore** openfight.js for now, it can be updated in a later version.

**Comment by Akahadaka** [comment]
Let me know if you need help with this. I'd suggest tackling it while still relatively fresh in your mind.

**Comment by goodwilltyping** [comment]
Yeah. it'd be like moving master to version 2. Would you think that the Json should go remote, while I'm copying the look at feel?

**Comment by Akahadaka** [comment]
We'll work on the dual-json task for version 2. Currently remote-json actually fills the dual-json role by using the browsers ability to do its own local storage. The reason we don't use a physical file for this is JavaScript's inability to save data to the users device directly as that would be a security issue.
